src/DistributedLock.Core/Internal/Data/DatabaseCommand.cs

```diff
@@ -183,7 +181,7 @@ private ValueTask PrepareIfNeededAsync(CancellationToken cancellationToken)
             {
                 return dbCommand.PrepareAsync(cancellationToken).AsValueTask();
             }
-#elif !NETSTANDARD2_0 && !NET461
+#elif !NETSTANDARD2_0 && !NET462
```

Comment:
Let's use `!NETFRAMEWORK` here and in similar situations we're touching when the relevant API isn't available in any NETFRAMEWORK version.